src/bmi_cfe.c

```diff
@@ -152,8 +152,11 @@ Variable var_info[] = {
     { 84, "Schaake_adjusted_magic_constant_by_soil_type",   "double", 1},
     { 85, "a_Xinanjiang_inflection_point_parameter",        "double", 1},
     { 86, "b_Xinanjiang_shape_parameter",                   "double", 1},
-    { 87, "x_Xinanjiang_shape_parameter",                   "double", 1}
+    { 87, "x_Xinanjiang_shape_parameter",                   "double", 1},
     //---------------------------------------
+    /*Soil Freeze-thaw model paramaters*/
+    { 88, "soil__ice_fraction_schaake",			    "double", 1}, //should it be here?? I don't see other input/outut variables in this list- AJ
```

Comment:
This `var_info` struct was introduced to help simplify bmi implementation & development; essential for [de/]serialization where all state variable are exposed.  This master version doesn't fully apply the structure, however.  See [owp-bmi-enhance](https://github.com/NOAA-OWP/cfe/blob/owp-bmi-enhance/include/bmi_cfe.h#L14) branch for a more proper, fully utilized, implementation of `Variable{}` struct. 

tl;dr  - This is fine for now

Reply:
For consistency, and ease of future serialization integration, can you please add these two soil-freeze-thaw params to [`get_state_var_ptrs()` ](https://github.com/NOAA-OWP/cfe/blob/1efb18ca63fca17e7a07782024df47528ff1dd14/src/bmi_cfe.c#L1805)? -thanks!
```
   //---------------------------------------
    /*Soil Freeze-thaw model paramaters*/
    ptr_list[88] = &(state->soil_reservoir.ice_fraction_schaake);
    ptr_list[89] = &(state->soil_reservoir.ice_fraction_xinan );
```

Reply:
DONE